Assets/cqui_settingselement.xml

```diff
@@ -119,8 +116,19 @@
                                     </Stack>
                                     <Stack Anchor="R,T" StackGrowth="Left" Offset="0,5">
                                         <GridButton ID="SmartbannerDistrictsAvailableCheckbox" Anchor="L,C" Offset="0,0" Size="300,24" Style="CheckBoxControl" String="LOC_CQUI_CITYVIEW_SMARTBANNER_DISTRICTS_AVAILABLE" />
-                                    </Stack>
-                                </Stack>
+                                    </Stack>                                
+									<Stack Anchor="R,T" StackGrowth="Left" Offset="0,5">
+										<GridButton ID="ShowSuzerainInCityStateBanner" Anchor="L,C" Offset="0,0" Size="300,24" Style="CheckBoxControl" String="LOC_CQUI_SHOW_SUZERAIN_IN_CITYSTATE_BANNER" />
+									</Stack>
+									<Stack ID="CityStateSuzerainOptions" Anchor="R,T" StackGrowth="Down">
+										<Stack Anchor="R,T" StackGrowth="Left" Padding="5">
+											<PullDown ID="CityBannerSuzerainStyle" Style="PullDownBlue" ScrollThreshold="400" Size="300,24" Offset="0,0" SpaceForScroll="0"/>
+										</Stack>
+										<Stack Anchor="R,T" StackGrowth="Left" Padding="5">
+											<GridButton ID="ShowSuzerainLabelInCityStateBanner" Anchor="L,C" Offset="0,0" Size="300,24" Style="CheckBoxControl" String="LOC_CQUI_LABELS_ON_SUZERAIN_ICON" />
+										</Stack>
+									</Stack>
+								</Stack>
```

Comment:
ShowSuzerainInCityStateBanner is now a pulldown with a label above it, and move the War Icon option here as well.
Note: putting the label on the left, like other places with the CQUI settings, causes all the things here to be pinned to the right side and it looks bad...  that can be fixed by setting the anchor to everything as "C,T", but that makes for a lot more changes in this file..

```suggestion
                                    <Stack Anchor="R,T" StackGrowth="Down" Offset="0,5" StackPadding="5">
                                        <Label Anchor="C,C" Style="ShellOptionText" String="LOC_CQUI_SHOW_SUZERAIN_IN_CITYSTATE_BANNER"/>
                                        <PullDown ID="ShowSuzerainInCityStateBanner" Style="PullDownBlue" ScrollThreshold="400" Size="300,24" Offset="0,0" SpaceForScroll="0"/>
                                    </Stack>
                                    <Stack ID="CityStateSuzerainOptions" Anchor="R,T" StackGrowth="Down" Offset="0,5">
                                        <Stack Anchor="R,T" StackGrowth="Left" Padding="5">
                                            <GridButton ID="ShowSuzerainLabelInCityStateBanner" Anchor="L,C" Offset="0,0" Size="300,24" Style="CheckBoxControl" String="LOC_CQUI_LABELS_ON_SUZERAIN_ICON" />
                                        </Stack>
                                    </Stack>
                                    <Stack Anchor="R,T" StackGrowth="Left" Padding="5" Offset="0,5">
                                        <GridButton ID="ShowWarIconInCityStateBanner" Anchor="L,C" Offset="0,0" Size="300,24" Style="CheckBoxControl" String="LOC_CQUI_SHOW_WAR_ICON_IN_CITYSTATE_BANNER" />
                                    </Stack>
```